recipes/hover/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "hover" %}
+{% set version = "0.5.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hover-{{ version }}.tar.gz
+  sha256: 2f84d974b13cbe2ae857782e1a18f2f40404ad750df87edde08cb99e3a44d9e0
+
+build:
+  number: 0
+  skip: true  # [win]
```

Comment:
Is there a reason why this is not being built on Windows?

Actually, shouldn't this be a [noarch: python](https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-builds) recipe?

Reply:
hover is pure-Python. This is not built on Windows because some of the dependencies ([`pytorch`](https://anaconda.org/conda-forge/pytorch) for example) has no Windows distribution.

Reply:
> A package is still considered noarch even if one of its dependencies is not available on a given platform.

Reply:
I see. Perhaps a silly question, but what's the appropriate method for a noarch package to pass the build on a platform where one of the deps is missing (at least missing from conda-forge)?

Reply:
Right, the tests will fail on Windows. That's not an issue for a noarch package here. The windows build for a noarch package only runs here for staged-recipes. The actual package will then always be built on Linux. (I guess, you could add some `[not win]` markers in the test section to make it pass on Windows but it's not worth it.)

Reply:
Thanks for the clarification! Then noarch is correct here. 

The upstream for this version (0.5.0) has been around for some time on PyPI. It's probably better to loose the requirement constraints for the next version. So I'll close this PR and start a new one for the new version.